src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -647,6 +647,9 @@ public Void caseTemplatizedType(TemplatizedType type) {
             emit("NodeList");
             return null;
           }
+          if (type.getDisplayName().equals("IThenable")) {
```

Comment:
Yeah, it's a bit ugly, but then again in my experience a bunch of special cases in code don't get nicer just because you move them out to a config file, you're just making the code slightly harder to understand through the indirection. Assuming we never have to actually configure this, I'd rather have this straight forward hard coded.